libparsec/crates/types/src/protocol.rs

```diff
@@ -113,8 +113,22 @@ impl TryFrom<&str> for ApiVersion {
  * ProtocolRequest
  */
 
+#[derive(Debug, Clone, Copy)]
+pub enum ProtocolFamily {
```

Comment:
This is because `ProtocolRequest` is the main type being defined, `ProtocolFamily` is just an inner type ("inner" meaning it is only used by `ProtocolRequest` even if it is public)